packages/firebase_database/pubspec.yaml

```diff
@@ -14,6 +14,8 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
+  firebase_core:
+    path: ../firebase_core
```

Comment:
Maybe I don't really understand how pub works. Will this retrieve both dependencies via pub?

Reply:
Just a placeholder value until the initial firebase_core is published, then it can be changed to `^0.0.1`